docs/visual-basic/misc/bc30249.md

```diff
@@ -45,7 +45,7 @@ A `Mid` statement or a `#Const` compilation directive does not include an assign
 -   Add an equal sign (`=`) and a value to be assigned to the string or the compilation constant.  
   
 ## See Also  
- [NotInBuild:Declaration Statements in Visual Basic](http://msdn.microsoft.com/en-us/81f3c398-f45c-4d95-80bf-aa39d1a0fb30)   
+ [Declaration Statements](~/docs/visual-basic/programming-guide/language-features/statements.md#declaration-statements)   
  [Mid Statement](../../visual-basic/language-reference/statements/mid-statement.md)   
  [#Const Directive](../../visual-basic/language-reference/directives/const-directive.md)   
  [NotInBuild:Assignment Statements](http://msdn.microsoft.com/en-us/eb4f91e9-fbbf-45ca-b21d-e8ae069de4f9)
```

Comment:
I'm changing to https://docs.microsoft.com/en-us/dotnet/articles/visual-basic/language-reference/operators/assignment-operators which seems to be the only topic on this. Unless you have a better replacement?